mmdet3d/core/bbox/structures/utils.py

```diff
@@ -142,3 +142,44 @@ def points_cam2img(points_3d, proj_mat):
     point_2d = torch.matmul(points_4, proj_mat.t())
     point_2d_res = point_2d[..., :2] / point_2d[..., 2:3]
     return point_2d_res
+
+
+def mono_cam_box2vis(cam_box):
+    """This is a post-processing function on the bboxes from Mono-3D task. If
+    we want to perform projection visualization, we need to:
```

Comment:
This is mainly because Nus 3D boxes have three rotation angles `(yaw, pitch, roll)`, but we assume the later two are `\pi / 2` and `0` and only predict `yaw`.